dictionaryutils/__init__.py

```diff
@@ -239,3 +239,19 @@ def resolve_schema(self, obj, root):
             return [self.resolve_schema(item, root) for item in obj]
         else:
             return obj
+
+    def allow_nulls(self):
+        for key, value in self.schema.items():
+            try:
+                try:
+                    required = value["required"]
+                except KeyError:
+                    required = []
+                for k, v in value["properties"].items():
```

Comment:
could you rename `k` and `v` to something more transparent to make it easier to read? maybe `prop_id`, `prop`